db_proposal.md

```diff
@@ -0,0 +1,34 @@
+Database Layout Proposal
+========================
+This is my proposal for the db layout.  Thoughts?
+
+Company
+========
+* Ticker (string, unique)
+* Name (string)
+* Industry (string)
+* Sector (string)
+
+PricePoint (historical, current)
+===========
+* Company::Ticker (foregin key, has_one)
+* Price (number)
+* Datetime (datetime, utc?)
+
+User
+====
+* name
+* email
+* password
+* favorited stocks (foreighn key to Company::Ticker, has_many)
+
+Tweet (current)
+=====
+* content (string)
+* author (string) [MAYBE?]
```

Comment:
I was looking into users favoriting stocks, I think we might need another table to hold the relation ship. Its a many-many relation ship (users can favorite multiple companies and companies can be saved my many users). We need another table ('favorites' or 'UserCompany', something else?)  to map userid companyid.